src/stores/auth_mall.js

```diff
@@ -18,7 +18,7 @@ class AuthMall {
 
   isUserNew = () => {
     let cookie_value = Cookies.get('type');
-    return (cookie_value === 'true') || (cookie_value !== undefined);
+    return (cookie_value == 'true');
```

Comment:
I am still seeing this error on this line when I start the frontend server:

```
src/stores/auth_mall.js
  Line 21:26:  Expected '===' and instead saw '=='  eqeqeq
```

I'll look into this a bit, but just wanted to highlight here. Will review what the diff is in JS.

Reply:
For ref: https://eslint.org/docs/rules/eqeqeq

Reply:
`==` is a soft comparison where the data type isn't considered. `===` checks the equality AND the data type. I think it's just expecting it to be `===` because I'm using `==` where both of the data types are strings and it's just saying 'might as well use `===` here'

Reply:
I think it's actually saying you NEED to use the `===` because the string "false" is == true in JS possibly. ( true == 'false' but NOT true === 'false ). Anyway, I think we're on the same page here. So leaving as resolved.